doc/manual/using_interactivity.rst

```diff
@@ -0,0 +1,239 @@
+Datasets and interactivity
```

Comment:
Is that broad enough to cover the inclusion of MonInj and the RTIO analyzer? Maybe technically, but not necessarily in a way that's intuitive to the user .... but I'm not happy with the 'interactivity' phrasing either, ideas I was toying with were: 

1. Some variant of 'going deeper with the management system' or 'extensions of the management system', i.e. making the page explicitly a miscellany of 'other features ARTIQ also has' 
2. Something like 'datasets and controllers' -- since one connecting feature of the ctlmgr/MonInj/analyzer sections are they all have something to do with controllers -- but that's really an internal detail and not title material 
3. The present use of 'interactivity' -- to collect these features as 'other ways of interacting with the core device besides running experiments' -- which to me seems most intuitive but I'm also dissatisfied that it doesn't really explain the presence of the ctlmgr section 

In terms of accuracy I favor \#1. Stylistically I kept trying to dodge it because I couldn't make it sound good. :thinking:

Reply:
I agree, but I don't think there's enough material for a dedicated section. How about this? 'Subsystems' is very vague but it covers the idea of the browser + controller ecosystem + Analyzer/Waveform + MonInj/rtiomon as available tools and relatively discrete elements of the infrastructure, which aren't strictly necessary for the core workflow.

Reply:
Not sure how clear the diffs will be but besides the name change the only significant edit is moving the sections on arguments back to the 'getting started' page.

Reply:
This is admittedly written as a tutorial currently... there's nothing preventing it from being rewritten as a description and moved out of the 'tutorial' section, but it'd also be retreading a lot of ground. 

I do think there's some value to a tutorial format for these things too? As it stands the page is also basically covering "all the _other_ tabs of the dashboard" (Datasets / MonInj / Waveform / Shortcuts, after Explorer and Args), which IMO has its intuitive value too. Instead of grouping things by functionality, it's grouping them in the order the user is likely to encounter them and wonder about them. Then once they've grasped how these tools work hopefully the actual appropriate use cases can make themselves obvious.

Reply:
Went for 'Data and user interfaces' (a little more cohesive to my ear than 'Datasets and user interfaces').